Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.cs

```diff
@@ -45,9 +48,102 @@ public partial class InfiniteCanvas : Control
         private InkToolbar _inkCanvasToolBar;
         private Canvas _mainContainer;
         private ScrollViewer _infiniteCanvasScrollViewer;
+        private StackPanel _canvasToolbarContainer;
+
+        /// <summary>
+        /// Gets or sets the width of the canvas, default value is the max value 2097152
+        /// </summary>
+        public double CanvasWidth
+        {
+            get { return (double)GetValue(CanvasWidthProperty); }
+            set { SetValue(CanvasWidthProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="CanvasWidth"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty CanvasWidthProperty =
+            DependencyProperty.Register(
+                nameof(CanvasWidth),
+                typeof(double),
+                typeof(InfiniteCanvas),
+                new PropertyMetadata(LargeCanvasWidthHeight, CanvasWidthHeightPropertyChanged));
+
+        /// <summary>
+        /// Gets or sets the height of the canvas, default value is the max value 2097152
+        /// </summary>
+        public double CanvasHeight
+        {
+            get { return (double)GetValue(CanvasHeightProperty); }
+            set { SetValue(CanvasHeightProperty, value); }
+        }
 
-        // TODO as property
-        internal const float LargeCanvasWidthHeight = 1 << 21;
+        /// <summary>
+        /// Identifies the <see cref="CanvasHeight"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty CanvasHeightProperty =
+            DependencyProperty.Register(
+                nameof(CanvasHeight),
+                typeof(double),
+                typeof(InfiniteCanvas),
+                new PropertyMetadata(LargeCanvasWidthHeight, CanvasWidthHeightPropertyChanged));
+
+        /// <summary>
+        /// Gets or sets the visibility of the toolbar.
+        /// </summary>
+        public Visibility ToolbarVisibility
```

Comment:
Would this normally be `IsToolbarVisible` as a bool?  Thinking of similar things already in the OS like the Ink Ruler: https://docs.microsoft.com/en-us/uwp/api/windows.ui.input.inking.inkpresenterruler.isvisible